src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -271,6 +272,19 @@ const AppWrapper = (props: { children: ReactNode }) => {
     }
   }, [isOnline, authentication.loggedIn, isMessagesHandled, dispatch]);
 
+  useEffect(() => {
+    if (isInitAppSuccess) {
+      if (!authentication.loggedIn) {
+        Agent.agent.signifyNotifications.stopNotification();
+      } else {
+        Agent.agent.signifyNotifications.startNotification();
+        Agent.agent.signifyNotifications.onNotificationStateChanged((event) => {
```

Comment:
If we do my above comment, then we can move this `onNotificationStateChanged` call back to where it was (just call it once, so we have one loop that is continuing or fetching)